R/adjacent.R

```diff
@@ -762,11 +760,15 @@ transitioncalc <- function(classimgdat, colornames) {
   }
```

Comment:
I'm pretty sure the for loops above can be removed. I will have a look whenever I get the chance.

Reply:
Yep — that's a work-in-progress sorry, should've commented as such. I'm playing with the ability for users to give colour-classes informal 'names' ('red', 'green'...), so transitions would be summarised as red_green, green_blue etc.  instead of 1_2, 2_3. Not quite happy with it atm, so put it on hold for now.